### ISSUE TITLE
Delineate between staff and volunteers

### ISSUE THREAD

**Issue by jmdawson** [MidwestFurryFandom/rams#146]
> I’m wondering if you know if there is any programmatic uses for the ribbons or if they are just flags that have no affect on how a registration record is treated?
> 
> Here’s what I’m running into…how to mark staff members in the registration database. While we do have that “Staff” badge type, the problem is that the badge type would change if the staff member upgrades to a sponsor or shiny level. Magfest doesn’t run into this problem because there is only one badge type and they do it by kickins. We can’t use the volunteering checkbox because that comes from the form a normal attendee would use. I noticed there is a “Department Head” ribbon type already there, and I’m wondering if that ties into anything specific or if I can co-op it as my staff flag. If that ribbon setting means something, I’m guessing we will need to add a specific “Staff” ribbon type. It doesn’t have to be done right now, as we won’t really be dealing with staff until after Labor Day.

**Comment by kitsuta** [comment]
Yeah, I'd recommend adding a Staff ribbon type via config. Many of the default ribbons 'do stuff' (otherwise we'd take them out of the system as extra cruft). In the case of the department head ribbon, it makes people 'trusted' in any department they're assigned to.
